logger_test.go

```diff
@@ -133,7 +133,7 @@ func TestLogger(t *testing.T) {
 		rest := str[dataIdx+1:]
 
 		// This test will break if you move this around, it's line dependent, just fyi
-		assert.Equal(t, "[INFO]  go-hclog/logger_test.go:146: test: this is test: who=programmer why=\"testing is fun\"\n", rest)
+		assert.Equal(t, "[INFO]  go-hclog/logger_test.go:129: test: this is test: who=programmer why=\"testing is fun\"\n", rest)
```

Comment:
the calling loc here seems to be 129 here so not sure why it was 146 earlier https://github.com/hashicorp/go-hclog/pull/51/files#diff-0d02175655a9ca1a852edb40e80012b1R129